docs/demos/filters/min_max_filter.jl

```diff
@@ -0,0 +1,56 @@
+# Topic : Max Min Filters
+# reference paper : https://core.ac.uk/download/pdf/194053536.pdf
+
+
+# In this tutorial we see how can we can effectively use max and min filter to distinguish 
+# between ramp("smooth") and ripple("texture") edges of B&W image.We will using the mapwindow function of
```

Comment:
(Not important) word choice: `B&W image` => `grayscale image`

Reply:
`ramp("smooth")` and `ripple("texture")` looks really like a function call 😂 

How about "between the ramp/smooth and ripple/texture part of the grayscale image" ?

Reply:
I was actually trying to keep a close correspondence to the paper :sweat_smile: .Anyway I had to remove those slashes too because it turns out demo cards can't render slashes.